client/src/components/Drawer/Drawer.tsx

```diff
@@ -8,6 +8,7 @@ import * as S from './Drawer.style';
 
 interface DrawerProps {
   isOpen: boolean;
+  myProfile: Friend | null;
```

Comment:
해당 타입이 범용적으로 사용되는 경우 타입명을 `User`로 변경하면 좋을 것 같습니다.
```suggestion
  myProfile: User | null;
```